index.js

```diff
@@ -34,6 +39,22 @@ var cli = new Liftoff({
   completions: completion,
   extensions: interpret.jsVariants,
   v8flags: v8flags,
+  configFiles: {
+    '.gulprc': {
+      home: {
+        path: '~',
+        extensions: interpret.extensions,
+      },
+      project: {
+        path: '.',
+        extensions: interpret.extensions,
+      },
+      cwd: {
+        path: process.env.INIT_CWD,
+        extensions: interpret.extensions,
```

Comment:
I mistook. When `gulpfile.js` is not found, `env.cwd` become same with `process.env.INIT_CWD`. So this `cwd` property is not needed. I'll remove this property.
(And It would be better to rename the key name `project` to `cwd`.)